validate/vm.go

```diff
@@ -23,13 +23,13 @@ type mockVM struct {
 
 	code *bytes.Reader
 
-	polymorphic bool    // whether the base implict block has a polymorphic stack
+	polymorphic bool    // whether the base implicit block has a polymorphic stack
 	blocks      []block // a stack of encountered blocks
 
 	curFunc *wasm.FunctionSig
 }
 
-// a block reprsents an instruction sequence preceeded by a control flow operator
+// a block represents an instruction sequence proceeded by a control flow operator
```

Comment:
`s/proceeded/preceded/`

Reply:
Oh good point.  I got that one wrong too. :wink:

Just force pushed an updated commit with that fixed.